jest.config.js

```diff
@@ -0,0 +1,27 @@
+/** @type {import('ts-jest').JestConfigWithTsJest} */
+module.exports = {
+  // preset: 'ts-jest',
+  // testEnvironment: 'node',
```

Comment:
do these comments do anything? should we remove?

Reply:
Done.